frontend/src/app/shared/custom/scoring/scoring.component.ts

```diff
@@ -127,16 +145,16 @@ export class ScoringComponent implements OnInit, AfterViewInit, OnChanges {
       this.labelPercentage = of(percentage);
       if (percentage < 30) {
         this.stretched = false;
-        this.failPercent = (100 / 30) * percentage;
+        this.failPercent = 100 / 30 * percentage;
```

Comment:
i would leave the parenthesis just because i do not trust ts. You are right theoretically the calculation should remain the same but as we all know ts sometimes does some fancy stuff...

Reply:
lint automatically removes them